org.gnome.Evolution.json

```diff
@@ -0,0 +1,381 @@
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.27.tar.bz2",
+					"sha256": "4f93aac6fecb7da2b92871bb9ee33032be6a87b174f54abf8ddf0911a22d29d2"
+				}
+			]
+		},
+
+		{
+			"name": "libgcrypt",
+			"cleanup": [
+				"/bin",
+				"/share/info",
+				"/share/man"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.1.tar.bz2",
+					"sha256": "7a2875f8b1ae0301732e878c0cca2c9664ff09ef71408f085c50e332656a78b3"
+				}
+			]
+		},
+
+		{
+			"name": "gnome-keyring",
+			"config-opts": [
+				"--with-pkcs11-config=/app/share/gk-to-remove",
+				"--with-pkcs11-modules=/app/share/gk-to-remove",
+				"--disable-doc",
+				"--disable-pam",
+				"--disable-p11-tests",
+				"--disable-ssh-agent"
+			],
+			"cleanup": [
+				"/etc/xdg",
+				"/share/gk-to-remove",
+				"/share/GConf"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/gnome-keyring/3.20/gnome-keyring-3.20.1.tar.xz",
+					"sha256": "97964e723f454be509c956ed5e38b5c2fd7363f43bd3f153b94a4a63eb888c8c"
+				}
+			]
+		},
+
+		{
+			"name": "evolution-data-server",
+			"buildsystem": "cmake",
+			"cleanup": [ "/share/GConf" ],
+			"config-opts": [
+				"-DENABLE_FILE_LOCKING=fcntl",
+				"-DENABLE_DOT_LOCKING=OFF",
+				"-DENABLE_OAUTH2=ON",
+				"-DENABLE_GTK=ON",
+				"-DENABLE_UOA=OFF",
+				"-DENABLE_GOA=ON",
+				"-DENABLE_EXAMPLES=OFF",
+				"-DENABLE_INTROSPECTION=OFF",
+				"-DENABLE_VALA_BINDINGS=OFF",
+				"-DENABLE_INSTALLED_TESTS=OFF",
+				"-DENABLE_GTK_DOC=OFF",
+				"-DENABLE_DBUS_SESSION_TOOL=ON",
+				"-DWITH_PRIVATE_DOCS=OFF",
+				"-DWITH_PHONENUMBER=OFF",
+				"-DWITH_SYSTEMDUSERUNITDIR=OFF"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.gnome.org/GNOME/evolution-data-server.git",
+					"branch": "gnome-3-30"
+				}
+			]
+		},
+
+		{
+			"name": "gtkspell3",
+			"config-opts": [
+				"--disable-gtk-doc",
+				"--disable-introspection",
+				"--disable-vala"
+			],
+			"cleanup": [ "/share/gtk-doc" ],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://downloads.sourceforge.net/project/gtkspell/3.0.9/gtkspell3-3.0.9.tar.xz",
+					"sha256": "a4f4a4a2789f7499563e26d96b22d8085222ebe278da47d026b2de782b8b4d26"
+				}
+			]
+		},
+
+		{
+			"name": "evolution",
+			"buildsystem": "cmake",
+			"config-opts": [
+				"-DENABLE_AUTOAR=OFF",
+				"-DENABLE_CONTACT_MAPS=OFF",
+				"-DENABLE_GNOME_DESKTOP=OFF",
+				"-DENABLE_GTK_DOC=OFF",
+				"-DENABLE_INSTALLED_TESTS=OFF",
+				"-DENABLE_PST_IMPORT=OFF",
+				"-DENABLE_TEXT_HIGHLIGHT=OFF",
+				"-DENABLE_YTNEF=OFF",
+				"-DWITH_GLADE_CATALOG=OFF",
+				"-DWITH_HELP=OFF"
+			],
+			"cleanup": [
+				"/etc/xdg",
+				"/share/GConf"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.gnome.org/GNOME/evolution.git",
+					"branch": "gnome-3-30"
```

Comment:
Is this a tag or a branch? If the latter, specify also a commit hash for builds reproducibility.